packages/node-app/src/routes/giveaway.route.ts

```diff
@@ -26,11 +26,10 @@ const upload = multer({ storage });
 export const router = express.Router();
 
 router.get('/', listGiveaways);
+router.post('/', verifyAdmin, upload.single('image'), createGiveaway);
 router.get('/:id', getGiveaway);
+router.put('/:id', verifyAdmin, upload.single('image'), updateGiveaway);
 router.put('/:id/participants', addParticipant);
-router.get('/:id/participants', getParticipants);
+router.get('/:id/participants', verifyAdmin, getParticipants);
```

Comment:
Already discussed on the previous PR, but I will write this comment as a warning, just to make sure this is not pushed to the main branch